autopts/wid/micp.py

```diff
@@ -68,10 +68,12 @@ def hdl_wid_5(params: WIDParams):
     stack = get_stack()
     gain = random.randrange(101, 127, 1)
     btp.aics_set_gain(gain, addr_type, addr)
-    ev = stack.aics.wait_aics_state_ev(addr_type, addr, 30)
-    # TODO: Should return False if ev is None
-
-    return True
+    ev = stack.aics.wait_aics_procedure_ev(addr_type, addr, 10)
+    if ev[3] == 2:
+        # opcode == 0x02: set gain
+        return True
+    else:
+        return False
```

Comment:
```suggestion
    return ev[3] == 2
```
Short the other ones too.